### PR TITLE
Added Github link to this section :D

### PR DESCRIPTION
**Pull request by Beta-King**
http://rawgit.com/Beta-King/labyrinth/master/index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,12 @@
 									<i class="fa fa-google-plus"></i>
 								</a>
 							</li>
+                            <li>
+								<a class="btn-floating transparent waves-effect waves-light github" target="_blank" href="https://github.com/fossasia"
+								class="social-icon">
```
Comment (niccokunzmann) [inline index.html]: Please do not use two class attributes but one and add social-icon to the end.

**Comment by niccokunzmann** [comment]
Thanks!

File: index.html (hunk 1 above)
Comment (niccokunzmann) [reply index.html]: #411 the html check fails otherwise

**Comment by Beta-King** [comment]
Ohkk sorry for that... thanks :D

**Comment by Beta-King** [comment]
#414 - solves this :D

**Pull request merged**